### ISSUE TITLE
Tabeli sortimine

### ISSUE THREAD

**Issue by RRisto** [hannatuus/EL-toetuste-saajad-ja-andjad#2]
Tabeli sortimisel on midagi lappama läinud. Kui vaadata 10 suurema saaja tabelit, siis value on järjekorras, kuid, value_tuhandetes ei ole:
toetuse_saaja_registri_kood     variable   value value_tuhandetes
1                     10000219 summa_saadud 2949220           3.9832
2                     10000356 summa_saadud 1478634           0.6871
3                     10000372 summa_saadud  855407           3.2138
4                     10000550 summa_saadud  707780           1.8473
5                     10000573 summa_saadud  626531           4.9777
6                     10000900 summa_saadud  589598           2.4904
7                     10000952 summa_saadud  452661           7.1387
8                     10001118 summa_saadud  369134           0.5747
9                     10001420 summa_saadud  363052           0.2257
10                    10001437 summa_saadud  339249           0.5293

Kas oskad arvata, kus võib viga olla?

**Comment by hannatuus** [comment]
value_tuhandetes tahtis ka sortimist

**Comment by RRisto** [comment]
kas toetuse saajad on siis õiges järjekorras? :)

**Comment by hannatuus** [comment]
Nope... ühesõnaga miskipärast pole saajad ja nende numbrid seotud, aga ma pole lahendust leidnud.
